src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -321,4 +323,23 @@ public static void verifyColumnDisplayOrder(String column1Name, String tableName
         assertTrue(expectedResult, table, () -> table.verifyColumnDisplayOrder(column1Name, column2Name));
     }
 
+    /**
+     * Verify the element exists in row which have stored value in column name
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>I verify row in the Project Files Table with value selected for the Search Input contains the xpath //i[contains(concat(' ', @class, ' '), ' fa-toggle-on ')]</li>
+     * </ul>
+     * @param tableName String the name of the table element
+     * @param key String the key used to retrieve the value
+     * @param xpath String xpath value to element to click
+     */
+    @Then("^I verify row in the (.*?) with value (?:entered|selected|used) for the (.*?) contains the xpath (.*?)$")
```

Comment:
I think this would read better if we added a "the" in front of "row" and "value". So `I verify the row in the _ with the value ...`. If you want to make them optional, that works too.